srcs/main.c

```diff
@@ -27,13 +27,34 @@
  */
 
 
-int	main(void)
+int	main(int ac, char *av)
 {
+	int	fd;
 	/* 1. assign signals to appropriate handlers
 	 * 2. infinite while loop
 	 *   a. get user input
 	 *   b. parse it
 	 *   c. execute it
 	 */
+	if (ac > 2)
+	{
+		// error message ?
+		return (1);
+	}
+	if (ac == 2)
+		fd = open(av[1], O_RDONLY); // should it be created if non-existent?
```

Comment:
I mean if we pass a file as an argument that doesn't exist. If we run ./minishell test_file but there is no test_file on that directory. I think we shouldn't have the trouble of creating it, cos if we do it will be empty thus will have no instructions, but I guess I wrote that when I was thinking about it

Reply:
ah, but then, what happens btw. I mean in the shell nothing happens if you press "space", "enter".
I'm not sure you need to create anything, just run  it as if it's empty ?